commands.py

```
@@ -0,0 +1,242 @@
+                            total_lines_processed += 1
+
+                    except json.JSONDecodeError:
+                        # Not JSON, try to extract simple key=value pairs
+                        if "=" in line and lines_in_file < 5:  # Only sample a few non-JSON lines
+                            _extract_simple_fields(line, field_info)
+                            lines_in_file += 1
+                            total_lines_processed += 1
+
+            files_processed += 1
+
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)